ipn/ipnlocal/cert.go

```diff
@@ -121,17 +136,21 @@ func (b *LocalBackend) GetCertPEM(ctx context.Context, domain string) (*TLSCertK
 	return pair, nil
 }
 
-func (b *LocalBackend) shouldStartDomainRenewal(cs certStore, domain string, now time.Time, pair *TLSCertKeyPair) (bool, error) {
+func (b *LocalBackend) shouldStartDomainRenewal(cs certStore, domain string, now time.Time, pair *TLSCertKeyPair) (renew bool, err error) {
 	renewMu.Lock()
 	defer renewMu.Unlock()
-	if last, ok := lastRenewCheck[domain]; ok && now.Sub(last) < time.Minute {
-		// We checked very recently. Don't bother reparsing &
-		// validating the x509 cert.
-		return false, nil
-	}
-	lastRenewCheck[domain] = now
+	if last, ok := lastRenewCheck[domain]; ok && now.Sub(last.time) < time.Hour {
```

Comment:
i think this is still too frequent, we get the SuggestedWindow from LE. We should cache until then IMO.